### DISCUSSION TITLE
Reuqest album items returns only first 20 tracks

### DISCUSSION THREAD

**Discussion by sonymaster** [tidal-music/discussions#101] [Report bug]
Hallo,

Calling the following url returns only the first 20 tracks. But the album has 34 tracks.
https://openapi.tidal.com/v2/albums/239938477/relationships/items?countryCode=DE&include=items

How can I get a list with all tracks?

**Comment by aliaksei-taliuk** [comment]
Hi @sonymaster 

There is "links" field in the doc root. Please use "next" link with the cursor pointing to the next page of 14 items. 

Regards, Aliaksei

**Reply by sonymaster** [reply]
That was the trick... many thanks!
